base/src/main/java/com/maubis/scarlet/base/note/formats/recycler/FormatTextViewHolder.kt

```diff
@@ -30,7 +30,7 @@ open class FormatTextViewHolder(context: Context, view: View) : FormatViewHolder
   protected val edit: EditText = root.findViewById(R.id.edit)
   protected val actionMove: ImageView = root.findViewById(R.id.action_move_icon)
 
-  protected var format: Format? = null
+  protected lateinit var format: Format
```

Comment:
Thanks!

Really hope this doesnt crash. I don't trust  recycler view lifecycles a lot. But #yolo

Reply:
It shouldn't cause any problem. The only difference is that now accessing format when it is uninitialized will throw a `UninitializedPropertyAccessException` instead of a `NullPointerException` (because you were using `!!` all the time).